internal/services/network/network_interface_resource_test.go

```diff
@@ -377,6 +377,28 @@ func TestAccNetworkInterface_pointToGatewayLB(t *testing.T) {
 	})
 }
 
+func TestAccNetworkInterface_tagsOfNicAttachedToPrivateEndpointCanBeUpdated(t *testing.T) {
```

Comment:
Can we add one more test case to cover the scenario where the update includes tag changes for a NIC that has no PE associated? Or please ignore this comment if there is already such test case.

Reply:
It is already covered on this existing test case: `TestAccNetworkInterface_tags`